src/anchorfm-pupeteer/index.js

```diff
@@ -59,23 +53,19 @@ async function selectYearInDatePicker(page, navigationPromise, year) {
     currentHighestYear = await page.$eval('tbody > tr:last-child > td:last-child', (e) => e.getAttribute('data-value'));
   }
 
-  const tdYear = await page.$(`tbody > tr > td[data-value="${year}"]`);
-  await tdYear.click();
+  await clickSelector(page, `tbody > tr > td[data-value="${year}"]`)
   await navigationPromise;
 }
 
 async function selectMonthInDatePicker(page, navigationPromise, month) {
-  const [tdMonth] = await page.$x(`//tbody/tr/td[contains(text(),"${month}")]`);
-  await tdMonth.click();
+  await clickSelector(page, `//tbody/tr/td[contains(text(),"${month}")]`);
   await navigationPromise;
 }
 
 async function selectDayInDatePicker(page, navigationPromise, day) {
   const dayWithRemovedZeroPad = parseInt(day, 10);
-  const tdDay = await page.$(
-    `tbody > tr > td[data-value="${dayWithRemovedZeroPad}"][class*="rdtDay"]:not([class*="rdtOld"]:not([class*="rtdNew"])`
-  );
-  await tdDay.click();
+  await clickSelector(page,
+    `tbody > tr > td[data-value="${dayWithRemovedZeroPad}"][class*="rdtDay"]:not([class*="rdtOld"]:not([class*="rtdNew"])`)
```

Comment:
Please add semi-colon at the end, i think eslint enforces that too.

Please add to all other places if missing.